### DISCUSSION TITLE
code-server —link background

### DISCUSSION THREAD

**Discussion by DaStormer** [coder/code-server#3305] [Q&A] [answered]
Is it possible to run `code-server --link` in the background?

**Comment by jsjoeio** [comment, marked as answer]
I thought you would be able to with `code-server --link &` but that doesn't work for me locally. cc @code-asher or @oxy any ideas?

**Reply by code-asher** [reply]
That should work; I gave it a try and it came up. Does it give any kind of error message?

**Reply by DaStormer** [reply]
This worked for me. Thank you!

**Reply by jsjoeio** [reply]
Oh! Maybe it was working and I killed the process lol glad it worked!

**Reply by Said113** [reply]
you can also run "bg" or "&"

**Reply by jsjoeio** [reply]
TIL about `bg`! Thanks for the tip @Said113 🙌

**Reply by bpmct** [reply]
You can also create a systemd service and simply run it with the --link flag. You'll need to monitor the logs to get the initial OAuth URL to link with GitHub, but then you're good to go
